apps/mobile/src/screens/auth/auth.tsx

```diff
@@ -1,9 +1,15 @@
 import React from "react";
 
+import { Planet } from "~/libs/components/background-wrapper/libs/components/planet/planet";
```

Comment:
```suggestion
import { Planet } from "~/libs/components/components";
```

Reply:
I don't think that we need this here at all. We'll revise BackgroundWrapper logic later

Reply:
remove Planet from auth screen